relay-server/src/actors/project_redis.rs

```diff
@@ -38,6 +39,20 @@ impl From<serde_json::Error> for RedisProjectError {
     }
 }
 
+fn parse_redis_response(raw_response: &[u8]) -> Result<ProjectState, RedisProjectError> {
+    let decoded = metric!(timer(RelayTimers::ProjectStateDecompression), {
+        zstd::decode_all(raw_response)
+    });
```

Comment:
Instead of checking for zstandard's magic number explicitly, I just run decompression here, assuming that the library already efficiently checks if the given blob is in zstandard format. We can use the logged metric to verify this.